support-docs/README.md

```diff
@@ -244,6 +244,220 @@ The table below gives a description of the properties used to specify a switchin
+|voutTyp|typical output voltage a device can regulate|unit| |
+|voutMin|minimum output voltage a device can regulate|unit| |
+|voutMax|maximum output voltage a device can regulate|unit| |
+|ioutTypPerString|typical output current per string a device can regulate|unit| |
+|ioutMinPerString|minimum output current per string a device can regulate|unit| |
+|ioutMaxPerString|maximum output current per string a device can regulate|unit| |
+|ioutAccuracyTyp|typical accuracy of per string current regulated by a device|unit| |
+|ioutAccuracyMin|minimum accuracy of per string current regulated by a device|unit| |
+|ioutAccuracyMax|maximum accuracy of per string current regulated by a device|unit| |
+|fswTyp|typical switching frequency of a device|unit| |
+|fswMin|minimum switching frequency of a device|unit| |
+|fswMax|maximum switching frequency of a device|unit| |
+|currentConsumption|current consumption of a device|array of currentConsumption| |
+|integratedFets|whether a device contains integrated switching mosfets|Boolean| |
+|integratedFetProperties|describes integrated fet current limits and rdson properties|powerFetProperities| |
+|currentMatchingAccuracyTyp|typical current matching between LED strings|unit| |
+|currentMatchingAccuracyMin|minimum current matching between LED strings|unit| |
+|currentMatchingAccuracyMax|maximum current matching between LED strings|unit| |
+|dimmingSupport|whether a device supports output current dimming|Boolean| |
+|dimmingControl|whether a device is dimmed by PWM or I2C|String| |
+|dimmingFrequencyTyp|typical dimming frequency of a device|unit| |
+|dimmingFrequencyMin|minimum dimming frequency of a device|unit| |
+|dimmingFrequencyMax|maximum dimming frequency of a device|unit| |
+|dimmingRatioTyp|typical dimming ratio of a device|ratio| |
+|dimmingRatioMin|minimum dimming ratio of a device|ratio| |
+|dimmingRatioMax|maximum dimming ratio of a device|ratio| |
+|componentProtectionThresholds|Thermal and power supply protection thresholds of a device|array of componentProtectionThresholds| |
+|efficiency|efficiency vs forward current|graphDefiniton| |
+|package|package size|package| |
+|pins|array of pin objects with associated properties|array of pinSpec| |
+
+####  3.10.6	 Battery Charger
+
+|Property|Description|JSON Data Type|Required?|
+|:----|:----|:----|:----|
+|componentID|common component identifying information, such as mpn|componentID|Yes|
+|chargerType|battery charger type|String| |
+|converterType|switching charger type|String| |
+|chargerTopology|type of battery charger topology (Narrow VDC vs Hybrid Power Boost)|String| |
+|batteryConfig|battery configuration supported by the device|array of String| |
+|batteryCellChemistry|battery cell chemistry supported by the device|array of String| |
+|inputPowerSource|input power source supported by the device|array of String| |
+|inputCurrentAccuracy|accuracy of input current when set|unit| |

```

Comment:
It looks like you changed up the script to strip out the "definitions.json#/" prefix (https://github.com/edatasheets/digital-datasheets/blob/main/support-docs/README.md#3171-mosfet). I think we should be consistent. I'm assuming you prefer to strip the prefix; it is admittedly verbose, so I could get behind it. I just want to make sure we were on the same page, and also so that we can decide to update the cases that have already been committed.

Reply:
Oh that's a mistake. I changed that for some other debug. Let me get the original version and fix that :)